controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServerConfig.java

```diff
@@ -35,4 +35,39 @@
      * @return The RPC port which has to be registered with the cluster used for external access.
      */
     Optional<Integer> getPublishedRPCPort();
+
+    /**
+     * Fetches the settings which indicates whether authorization is enabled.
+     *
+     * @return Whether this deployment has auth enabled.
+     */
+    boolean isAuthorizationEnabled();
+
+    /**
+     * Fetches the location of passwd file for the users.
+     * This uses UNIX password file syntax. It is used for default implementation of authorization
+     * and authentication.
+     *
+     * @return Location of password file.
+     */
+    String getUserPasswdFile();
```

Comment:
Since the user password file is a specific configuration item that is only valid in the the case of the default PravegaAuthHandler, I'm wondering if this would be best wrapped up as a class like AuthHandlerConfig or similar that might have some map of key value pairs.  In the case of the default PravegaAuthHandler impl., there would only be one thing, the userpasswordfile here mentioned.  In the case of other handler impl., it could be the IP to an auth server etc.